docs/user-guide.rst

```diff
@@ -296,6 +296,31 @@ Anitya provides several different versions schemes.
     recent scheme and the rest will be moved to bottom unsorted.
 
 
+Version Pattern
+===============
+
+The version pattern is used to extract and match version numbers from
+project releases. It allows for specifying the format of version numbers
+using regular expressions.
+
+Patterns -
```

Comment:
You should add empty line after this, otherwise the list will not work correctly.